ui/app/pages/settings/networks-tab/network-form/network-form.component.js

```diff
@@ -94,24 +93,37 @@ export default class NetworkForm extends PureComponent {
   }
 
   resetForm() {
-    const {
-      rpcUrl,
-      chainId,
-      ticker,
-      networkName,
-      blockExplorerUrl,
-    } = this.props
+    const { rpcUrl, ticker, networkName, blockExplorerUrl } = this.props
 
     this.setState({
       rpcUrl,
-      chainId,
+      chainId: this.getDisplayChainIdFromProps(),
       ticker,
       networkName,
       blockExplorerUrl,
       errors: {},
     })
   }
 
+  /**
+   * Should be called the chainId is set from the props value.
```

Comment:
There is a typo on line 109 "Should be called the chainId is set from the props value."

Reply:
Fixed in: a9584c3